src/model/HeatExchangerDesiccantBalancedFlowPerformanceDataType1.cpp

```diff
@@ -0,0 +1,1497 @@
+*  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+***********************************************************************************************************************/
+
+#include "HeatExchangerDesiccantBalancedFlowPerformanceDataType1.hpp"
+#include "HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl.hpp"
+
+#include "Model.hpp"
+#include "Model_Impl.hpp"
+#include "HeatExchangerDesiccantBalancedFlow.hpp"
+#include "HeatExchangerDesiccantBalancedFlow_Impl.hpp"
+
+#include <utilities/idd/IddFactory.hxx>
+#include <utilities/idd/IddEnums.hxx>
+#include <utilities/idd/OS_HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1_FieldEnums.hxx>
+#include "../utilities/units/Unit.hpp"
+#include "../utilities/core/Assert.hpp"
+
+namespace openstudio {
+namespace model {
+
+  namespace detail {
+
+    HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl::HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl(
+      const IdfObject& idfObject, Model_Impl* model, bool keepHandle)
+      : ResourceObject_Impl(idfObject, model, keepHandle) {
+      OS_ASSERT(idfObject.iddObject().type() == HeatExchangerDesiccantBalancedFlowPerformanceDataType1::iddObjectType());
+    }
+
+    HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl::HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl(
+      const openstudio::detail::WorkspaceObject_Impl& other, Model_Impl* model, bool keepHandle)
+      : ResourceObject_Impl(other, model, keepHandle) {
+      OS_ASSERT(other.iddObject().type() == HeatExchangerDesiccantBalancedFlowPerformanceDataType1::iddObjectType());
+    }
+
+    HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl::HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl(
+      const HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl& other, Model_Impl* model, bool keepHandle)
+      : ResourceObject_Impl(other, model, keepHandle) {}
+
+    const std::vector<std::string>& HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl::outputVariableNames() const {
+      static std::vector<std::string> result;
+      if (result.empty()) {
+      }
+      return result;
+    }
+
+    IddObjectType HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl::iddObjectType() const {
+      return HeatExchangerDesiccantBalancedFlowPerformanceDataType1::iddObjectType();
+    }
+
+    ModelObject HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl::clone(Model model) const {
+      // clone the operating modes is already handle in ModelObject_Impl::clone since they are ResourceObjects
+      // We don't do ParentObject_Impl::clone since it'll also CLONE the children...
+      return ModelObject_Impl::clone(model);
+    }
+
+    std::vector<ModelObject> HeatExchangerDesiccantBalancedFlowPerformanceDataType1_Impl::children() const {
+      std::vector<ModelObject> result;
+
+      return result;
+    }
```

Comment:
This object has no children. Remove these methods altogether from the cpp and _Impl.hpp